src/components/forms/stay-connected.js

```diff
@@ -83,4 +81,4 @@ class StayConnected extends React.Component {
   }
 }
 
-export default StayConnected
+export default StayConnected
```

Comment:
Looks great overall! This file needs a newline at the end. Super nitpicky, lol. Everything else is awesome!

Reply:
Thanks for looking it over. I'll get the newline added and merged in!